src/actions/CreateWalletActions.tsx

```diff
@@ -222,20 +222,23 @@ export function createAccountTransaction(
               category: 'Expense:' + sprintf(lstrings.create_wallet_account_metadata_category, createdWalletCurrencyCode),
               notes: sprintf(lstrings.create_wallet_account_metadata_notes, createdWalletCurrencyCode, createdWalletCurrencyCode, config.supportEmail)
             }
-            paymentWallet.saveTxMetadata(edgeTransaction.txid, currencyCode, edgeMetadata).then(() => {
-              navigation.navigate('walletsTab', { screen: 'walletList' })
-              setTimeout(() => {
-                Alert.alert(lstrings.create_wallet_account_payment_sent_title, lstrings.create_wallet_account_payment_sent_message)
-              }, 750)
-            })
+            paymentWallet
+              .saveTxMetadata(edgeTransaction.txid, currencyCode, edgeMetadata)
+              .then(() => {
+                navigation.navigate('walletsTab', { screen: 'walletList' })
+                setTimeout(() => {
+                  Alert.alert(lstrings.create_wallet_account_payment_sent_title, lstrings.create_wallet_account_payment_sent_message)
+                }, 750)
+              })
+              .catch(err => showError(err))
```

Comment:
Should the `catch` go on `saveTxMetadata`, so we still navigate to the right place, even if we fail to save the tx details?